aicsimageio/tests/writers/test_timeseries_writer.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from pathlib import Path
+
+import dask.array as da
+import numpy as np
+import pytest
+
+from aicsimageio import exceptions
+from aicsimageio.writers import TimeseriesWriter
+
+
+@pytest.mark.parametrize(
+    "array_creator", [lambda shape: np.random.rand(*shape), da.random.random]
+)
```

Comment:
Does this need to be a parameter? Could it be a fixture (or just a plain old function)?

Reply:
Hrmmmm. You have always been better at pytest stuff, mind explaining how a fixture or function would be used in place of this?